x-pack/plugins/apm/public/components/app/RumDashboard/URLFilter/URLSearch/SelectableUrlList.tsx

```diff
@@ -83,6 +85,10 @@ export function SelectableUrlList({
 }: Props) {
   const [darkMode] = useUiSetting$<boolean>('theme:darkMode');
 
+  const { urlParams } = useUrlParams();
+
+  const { searchTerm } = urlParams;
```

Comment:
As this is only being pulled in for setting the default value, would it be better to push that responsibility to `index` (which is also reading this) and have `index` pass a prop in, something like `initialValue` or `defaultValue`, so that there's one less place touching URL params (even if it is read-only)?